.github/workflows/checkbox-snap-beta-release.yml

```diff
@@ -18,7 +18,11 @@ jobs:
     env:
       SERIES: series_${{ matrix.type }}${{ matrix.releases }}
```

Comment:
Nitpick #1: You don't use this env everywhere it could be.

Nitpick #2: Actually `${{ matrix.type }}${{ matrix.releases }}` looks also on its own to be very recurrently used. Not sure it makes sense to create a very dense set of env vars, but `TYPE_RELEASE` perhaps on balance clearer.

Reply:
(woa the #2 PR on this repo is actually the deb workflow)

I don't know if the places where the macro is not used support it (for example path). I would play safe with this and we can change it once we are sure